### PR TITLE
Update autoresponse.json

### PR DESCRIPTION
**Pull request by PadowYT2**

### CODE REVIEW

**Comment by Aberdeener** [comment]
The only change I approve is removing the mention of the old default template. Other things are 3rd party and therefore should not be in our bot. If you could revert those changes I will merge. Thanks

**Comment by Aberdeener** [comment]
I think it would be best if we forward all support queries for 3rd party templates to their author, rather than having some support for them in our guild. This would prevent people from getting confused about where to ask questions

**Comment by PadowYT2** [comment]
welp ok

File: commands.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
       "type": "argdir",
       "hidden": false
     },
-    {
+    	{
```
Comment (Derkades) [inline commands.json:15]: please revert this

**Comment by PadowYT2** [comment]
hm? it is better to use ``` tho

**Comment by Aberdeener** [comment]
Thanks!

**Pull request merged**